corehq/apps/users/models.py

```diff
@@ -1139,6 +1139,33 @@ def get(stale, raise_if_none):
         else:
             return None
 
+    @classmethod
+    @skippable_quickcache(['username'], skip_arg='strict')
+    def get_deleted_by_username(cls, username, strict=True):
```

Comment:
it looks like nothing uses `strict=False` - so I'd just remove that argument and simplify the underlying function. just thinking of the yagni principle